api/custom_auth/oauth/views.py

```diff
@@ -35,6 +38,10 @@ def login_with_google(request):  # type: ignore[no-untyped-def]
         )
         serializer.is_valid(raise_exception=True)
         token = serializer.save()
+        if settings.COOKIE_AUTH_ENABLED:
+            return authorise_response(
+                serializer.user, Response(status=HTTP_204_NO_CONTENT)
```

Comment:
How about we obtain the user from the token object?

```suggestion
                token.user, Response(status=HTTP_204_NO_CONTENT)
```